From a07fd0cb7cab97bf3edb55e5e721397188c94e66 Mon Sep 17 00:00:00 2001
From: Elliott Sales de Andrade <quantum.analyst@gmail.com>
Date: Mon, 19 Jun 2017 20:52:12 -0400
Subject: [PATCH 2/6] Add build include directory to gtkdoc source paths.

This enables gtkdoc to produce documentation on files that were
generated, using configure_file, for example.
---
 mesonbuild/modules/gnome.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/mesonbuild/modules/gnome.py b/mesonbuild/modules/gnome.py
index 3c80a094..00e5a8cb 100644
--- a/mesonbuild/modules/gnome.py
+++ b/mesonbuild/modules/gnome.py
@@ -700,6 +700,8 @@ class GnomeModule(ExtensionModule):
                 for inc_dir in src_dir.get_incdirs():
                     header_dirs.append(os.path.join(state.environment.get_source_dir(),
                                                     src_dir.get_curdir(), inc_dir))
+                    header_dirs.append(os.path.join(state.environment.get_build_dir(),
+                                                    src_dir.get_curdir(), inc_dir))
             else:
                 header_dirs.append(src_dir)
 
-- 
2.13.0

